test/unit/test_loss_image.py

```diff
@@ -118,3 +118,32 @@ def test_get_config(self):
             name="LocalNormalizedCrossCorrelation",
         )
         assert got == expected
+
+
+class TestGlobalNormalizedCrossCorrelation:
+    @pytest.mark.parametrize(
+        "y_true,y_pred,shape,expected",
+        [
+            (0.6, 0.3, (3, 3), 1),
+            (0.6, 0.3, (3, 3, 3), 1),
+            (0.6, -0.3, (3, 3, 3), 1),
+            (0.6, 0.3, (3, 3, 3, 3), 1),
+        ],
+    )
+    def test_output(self, y_true, y_pred, shape, expected):
+
+        y_true = y_true * np.ones(shape=shape)
```

Comment:
I suggest to use tf.ones to be safe, as `np.ones` tend to define numpy arrays in float64 :)